### DISCUSSION TITLE
记几位高中同学 | /cn/2020/07/high-schoolmates/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1379] [Comments]
https://yihui.org/cn/2020/07/high-schoolmates/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
你仿佛看见了我读书时划的线。今年我确实好几次想起这篇文章。

（又及：我调整了一下这些故事的顺序，现在得按小标题交叉引用、不能按前后顺序了，所以我相应修改了一下你的引用）

> ——原帖发布于 2020-07-22 06:01:53

[a comment by an automated account is not shown]

**Comment by Sijiu** [comment]
纯真年代， 好多次想写一篇高中小说， 这里看到了。
